runners/google-cloud-dataflow-java/worker/src/test/java/org/apache/beam/runners/dataflow/worker/DataflowExecutionContextTest.java

```diff
@@ -184,4 +195,39 @@ public void testDataflowExecutionStateTrackerRecordsCompletedProcessingTimes()
     Assert.assertEquals(
         expectedMetadata.userStepName(), tracker.getActiveMessageMetadata().get().userStepName());
   }
+
+  @Test
+  public void testDataflowExecutionStateTrackerDoesNotRecordCompletedProcessingTimesForBatch()
```

Comment:
do we need to add a test to show that it does record for streaming?

Reply:
this is implicitly covered in the pre-existing tests the were written for the original CL